support/values.yaml

```diff
@@ -15,18 +15,20 @@ prometheus:
         # Deploy onto user nodes
         key: hub.jupyter.org_dedicated
         value: user
+      - effect: NoSchedule
+        # Deploy onto user nodes
+        key: k8s.dask.org_dedicated
+        value: worker
     updateStrategy:
       type: RollingUpdate
   pushgateway:
     enabled: false
-  rbac:
-    create: true
   server:
     resources:
       # Without this, prometheus can easily starve users
       requests:
         cpu: 0.2
-        memory: 768Mi
+        memory: 512Mi
```

Comment:
Can you comment on this adjustment?

Reply:
Right now at least, these are all just estimates. The hope is that we can measure resource use of these and adjust accordingly.